bin/rtdev-build-releases.sh

```diff
@@ -81,9 +81,9 @@ kerl()
 
     echo " - Building Erlang $RELEASE (this could take a while)"
     # Use the Basho-patched version of Erlang 
-    if [ "$RELEASE" == "R15B01" ]; then
+    if [ ! -z $(echo "$RELEASE" | grep "R15B01") ]; then
         BUILD_CMD="./kerl build git git://github.com/basho/otp.git $R15_TAG $BUILDNAME"
-    elif [ "$RELEASE" == "R16B02" ]; then
+    elif [ ! -z $(echo "$RELEASE" | grep "R16B02") ]; then
```

Comment:
This seemed to work before anyway.  Here the output of bash -x bin/rtdev-build-releases.sh
```bash
echo ' - Building Erlang R16B02 (this could take a while)'
 - Building Erlang R16B02 (this could take a while)
+ '[' R16B02 == R15B01 ']'
+ '[' R16B02 == R16B02 ']'
+ BUILD_CMD='./kerl build git git://github.com/basho/otp.git OTP_R16B02_basho9 erlang-R16B02'
```